packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -484,6 +484,11 @@ export class AmqpConnection {
             throw new Error('Received null message');
           }
 
+          if (rpcOptions.routingKey !== msg.fields.routingKey) {
```

Comment:
Can you just run prettier? It has alot of extra tab spacing

Reply:
Fixed formatting issue. Cheers!